rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1186,7 +1238,13 @@ private void adjustBalancesIfChangeOutputWasDust(BtcTransaction btcTx, Coin sent
         for (TransactionInput transactionInput : btcTx.getInputs()) {
             sumInputs = sumInputs.add(transactionInput.getValue());
         }
-        Coin change = btcTx.getOutput(1).getValue();
+
+        Coin change = Coin.ZERO;
+        try {
+            change = btcTx.getValueSentToMe(getActiveFederationWallet(false));
```

Comment:
Let receive the wallet by param, so we don't make this method dependant only on the active federation wallet

Reply:
Done